### ISSUE TITLE
Road signs verifications sometimes act as like button is pressed multiple times

### ISSUE THREAD

**Issue by kolen** [mapillary/mapillary_issues#2870]
### Basic information
* Platform: `Web`
* Version: ?
* System/Browser: `Chrome 65.0.3325.73`

### Steps to reproduce behavior

 1. Open signs verification UI (I tried [the one for icy road signs](https://www.mapillary.com/verification/p/warning-icy-road-ww))
 2. Press confirm/reject for few signs

### Expected behavior

When I press green/red button (reject/confirm) once, current image is marked as rejected/accepted and next image appears.

### Actual behavior

Sometimes, after pressing red/green button, it looks like button is pressed two and more times:

- Animated green/red overlay appears over photo
- Next photo appears
- Immediately after this green/red overlay appears again
- Another photo appears

### Additional information

No suspicious records in js console. Can't reproduce when Chrome dev tools is open, likely it affects timing.

**Comment by kolen** [comment]
Possibly this is this case recorded in console, two "interaction" events one after another just after 0.15 seconds:

![image](https://user-images.githubusercontent.com/24191/36391254-d4ca7580-15b6-11e8-8eac-792c7d25e7be.png)

Or is it because "Couldn't fetch thumbnails, skipping"? So second "interaction" is like pressing "skip" button automatically? Then it's ok but the problem is that it feels like I pressed button two times accidentally.

**Comment by gyllen** [comment]
@tobias-o ^^^

**Comment by tobias-o** [comment]
> Or is it because "Couldn't fetch thumbnails, skipping"? So second "interaction" is like pressing "skip" button automatically? Then it's ok but the problem is that it feels like I pressed button two times accidentally.

@kolen you are right, this happens when we can't fetch the image crop for some reason. Instead of showing an error, we just skip the image and try the next. 

I'll try to think of a better way to handle the UX in such cases. This behavior can be confusing.
